index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <div class="header">
+        <div class="firstrow">
+            <p class="logo">Header logo</p>
+            <div class="links">
+                <p class="link">header link one</p>
+                <p class="link">header link two</p>
+                <p class="link">header link three</p>
+            </div>
+        </div>
+        <div class="secondrow">
+            <div class="hero">
+                <h1 class="heading">
+                    <b>This website is awesome</b>
+                </h1>
+                <p class="sub-heading">
+                    This website has some subtext that goes here under the main title.
+                    it's a smaller font and the color is lower contrast
+                </p>
+                <button type="button" class="sign">Sign up</button>
+            </div>
+        </div>
+    </div>
+
+    <div class="middle-container">
+        <div class="text">
+            <h2><b> Some random information. </b></h2>
+        </div>
+        <div class="middle-images">
+            <div class="images">
+                <img src="https://m.media-amazon.com/images/I/31e9Y7Ob5wL._AC_.jpg" />
+                <p class="middle-text">random text about image</p>
+            </div>
+        </div>
+    </div>
+    <div class="quote-container">
+        <div class="quote-box">
+            <p class="quote-text">
+                This is some inspiring quote, or a testmonial from a customer. Maybe
+                it's filling up space or maybe people will actually read it.Who knows?
+                All I know is it looks nice.
+            </p>
+            <b class="quote-name"> -Thor,God of Thunder </b>
+        </div>
+    </div>
+    <div class="signup-container">
+        <div class="signup-box">
+            <div class="signup-box-text">
+                <b class="main-text"> Call to action! it's time! </b>
+                <p class="sub-text">
+                    Signup for our product by clicking that button over there!
+                </p>
+            </div>
+            <div class="signup-button">
+                <button type="button">Signup</button>
+            </div>
+        </div>
+    </div>
+    <div class="footer-container">
+        <p class="footer-content">Copyright © The Odin Project 2021</p>
+    </div>
+</body>
+
+</html>
```

Comment:
No new line at the end, make sure there is not red sign at the end of the file once you push your code.